class BusinessException(Exception):
    pass


class InternalErrorException(Exception):
    pass

